src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -67,6 +67,8 @@ public Command parseCommand(String userInput) throws ParseException {
         case DeleteCommand.COMMAND_WORD:
             if (arguments.trim().startsWith("-d")) {
                 return new DeleteDeliveryCommandParser().parse(arguments.trim());
+            } else if (arguments.trim().startsWith("-s")) {
+                return new DeleteSupplierCommandParser().parse(arguments.trim());
```

Comment:
if we create a check for delete commands that start with -d and -s, then i think we should remove the else statement at the end as the deletecommand is no longer being used. we should stick to just using the delivery and supplier commands